lightstep_incident_response/manifest.json

```diff
@@ -14,7 +14,12 @@
     "classifier_tags": [
       "Supported OS::Linux",
       "Supported OS::Mac OS",
-      "Supported OS::Windows"
+      "Supported OS::Windows",
+      "Category::Alerting",
+      "Category::Collaboration",
+      "Category::Issue Tracking",
+      "Category::Notification",
+      "Category::Incident response"
```

Comment:
I don't think this classifier tag exist. I think the closest one we have is `"Category::Incidents"`. Can we also move it up to keep the alphabetical order.

Reply:
Made this change, Please review and confirm